exceptions.py

```diff
@@ -0,0 +1,30 @@
+class GameDoesntExistError(Exception):
+  """is thrown when there's no game in a chat"""
+
+class GameAlreadyExistError(Exception):
+  """is thrown when there's already a game in a chat"""
+  pass
+
+class NotEnoughNkapError(Exception):
+  """is thrown when a user doesn't have enough nkap for an operation"""
+  pass
+
+class NotVerifiedError(Exception):
+  """is thrown when a user is not verified or banned"""
+  pass
+
+class LobbyClosedError(Exception):
+  """is thrown when a game lobby has been closed"""
+  pass
+
+class NotEnoughPlayersError(Exception):
+  """is thrown when a game is about to start but there's not enough players"""
+  pass
+
+class DeckEmptyError(Exception):
+  """a rare issue where the deck is empty"""
+  pass
+
+class DifferentSuitError(Exception):
+  """is thrown when a player tries to play a card of a different suit"""
+  pass
```

Comment:
Could you that precedent cleaning with all those classes ? "pass" instruction is not really needed here
Also, if you want to raise a specific message on these class object call... we can do something like:

```python
class DifferentSuitError(Exception):
    def __init__(self, message=" a player tries to play a card of a different suit"):
        super().__init__(message)
```
what do you think ?